src/blockscoutApi/types.ts

```diff
@@ -293,3 +293,48 @@ export interface BlockscoutTransactionResponseTxResult {
   txreceipt_status: string;
   value: string;
 }
+
+export interface NftTokenHolderOwner {
```

Comment:
We should reuse the Account interface, according to Blockscout docs, is reusing that interface.